justfile

```diff
@@ -4,4 +4,4 @@
 # Install all scripts in `bin/` to `~/.local/bin/`
 install:
     mkdir -p ~/.local/bin
-    ln -s $(pwd)/bin/* ~/.local/bin
\ No newline at end of file
+    ln -sf $(pwd)/bin/* ~/.local/bin
```

Comment:
What's the difference between `s` and `sf`?

Reply:
sf is `soft link, force overwrite`. Otherwise it throws an error if it already exists and that's annoying.